spec/spec.yml

```diff
@@ -17,6 +17,16 @@ produces:
 consumes:
   - application/json
 
+securityDefinitions:
+   # X-API-Key: abcdef12345
```

Comment:
What's this part about?

Reply:
It makes the spec play nice with swaggerhub so that https://app.swaggerhub.com/api/notthatbreezy/Raster-Foundry/1.18.0 plays nice